scripts/execute_unittest.sh

```diff
@@ -1,3 +1,4 @@
+set -e
```

Comment:
Is this a way to bubble error code up? Can you add explain action of what it does for my understanding as a response to this comment

Reply:
set -e is used for error exit. It aborts the script at first error, when a command exits with non-zero status

Reply:
Thanks.
For more details folks can run below on terminal `help -m set `